deployments/controller/dashboard/metrics/task_execution.yaml

```diff
@@ -0,0 +1,25 @@
+name: task_execution
+description: Execution details of a singular task executed by the controller 
+filter: |-
+  resource.type="k8s_container"
+  resource.labels.project_id="$REGISTRY_PROJECT_IDENTIFIER"
+  resource.labels.location="us-central1-a"
+  resource.labels.cluster_name="registry-backend"
+  resource.labels.namespace_name="default"
+  severity>=DEFAULT
+  textPayload=~"(Successful|Failed) Execution:"
+labelExtractors:
+  action: REGEXP_EXTRACT(textPayload, "action={([^})]*)}")
+  status: REGEXP_EXTRACT(textPayload, "(Successful|Failed) Execution:")
+  task_id: REGEXP_EXTRACT(textPayload, "taskID={([^}]*)}")
```

Comment:
If we're going to rely on the format of log messages for our dashboards, I think we should add some comments to the log messages that point out this dependency. Otherwise it's pretty easy for someone to change the message without noticing these dashboards rely on a specific format.

Reply:
Done.